tldr/cross_file_calls.py

```
@@ -2409,6 +2411,126 @@ def _get_php_class_context(node, source: bytes) -> str | None:
+    def _extract_module_name(call_node) -> str | None:
+        """Extract the module name from a defmodule call node."""
+        for child in call_node.children:
+            if child.type == "arguments":
+                for arg_child in child.children:
+                    if arg_child.is_named:
+                        text = _get_alias_text(arg_child)
+                        if text:
+                            return text
+        return None
+
+    def _extract_func_name(call_node) -> str | None:
```

Ruff (0.15.5)

[warning] Too many branches (13 > 12)

(PLR0912)